code/src/components/MessageForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const MessageForm = ({ messageNew, onMessageNewChange, onFormSubmit }) => {
+    return (
+        <form className="message-form" onSubmit={onFormSubmit}>
+            <label className="form-title" htmlFor="newMessage">What's making you happy right now ?</label>
+            <input
+                className="text-area"
+                id="newMessage"
+                type="text"
+                value={messageNew}
+                onChange={onMessageNewChange}
+                minLength="5"
```

Comment:
I think it's good you matched the length the form can accept with what the API can accept.